extensions/kubectl/template.go

```diff
@@ -63,7 +64,7 @@ func CreateJobAndRunKubectlCommands(clusterID, jobname string, job *batchv1.Job,
 
 	sa := &corev1.ServiceAccount{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "rancher-installer2",
+			Name: namegen.AppendRandomString("rancher-installer2"),
```

Comment:
lets change rancher-installer2 to kubectl-command or any similar wording?

Reply:
Changed 👍